### PR TITLE
[arb-mod] add Arabic subtitles for "why vue" video

### PR DESCRIPTION
**Pull request by chadidi**
This is the Arabic translation, but needs some review from message 17 and up cause it was translated with google translation, I just had the time to review the first 40 seconds.

any help appreciated

### CODE REVIEW

**Comment by sdras** [comment]
Thank you! I will ask around for help on the review here and get back to you when we have someone.

File: assets/why-vue/arabic.js.srt [hunk 1]
```diff
@@ -24,565 +24,513 @@
...
 00:00:19,781 --> 00:00:23,948
-JavaScript frameworks like,
-Angular, React, or Vue.
+أطر عمل جافا سكريبت مثل Angular أو React أو Vue.
 
 10
 00:00:24,821 --> 00:00:27,829
-Vue is an approachable,
-versatile, and performant
+Vue هو إطار جافا سكريبت ودود، متعدد الاستعمالات و بأداء
```
Comment (Ma7eer) [inline assets/why-vue/arabic.js.srt]: I wouldn't use وبأداءhere. 

Maybe we can try وفعال?

File: assets/why-vue/arabic.js.srt [hunk 2]
```diff
@@ -24,565 +24,513 @@
...
 00:00:29,860 --> 00:00:33,381
-a more maintainable,
-and testable code base.
+قاعدة شفرة أكثر قابلية للإصلاح وللإختبار.
 
 13
 00:00:33,381 --> 00:00:36,229
-Vue is a progressive JavaScript 
-framework, which means,
+Vue هو إطار عمل جافا سكريبت تقدمي، مما يعني،
```
Comment (Ma7eer) [inline assets/why-vue/arabic.js.srt]: I believe that before moving to the next line we need to add انه

the sentence would be: هو اطار عمل جافا سكريبت تقدمي مما يعني "انه" اذا كانت لديك... and so on

File: assets/why-vue/arabic.js.srt [hunk 3]
```diff
@@ -24,565 +24,513 @@
...
 
 14
 00:00:36,229 --> 00:00:38,621
-if you have an existing
-server-side application,
+إذا كان لديك تطبيق موجود من جانب الخادوم
```
Comment (Ma7eer) [inline assets/why-vue/arabic.js.srt]: I think a better word than خادوم which is completely wrong would be just the arabic equivalent: السيرفير

The sentence would be: اذا كان لديك تطبيق موجود من جانب السيرفير

File: assets/why-vue/arabic.js.srt [hunk 4]
```diff
@@ -24,565 +24,513 @@
...
 
 15
 00:00:38,621 --> 00:00:41,627
-you can plug Vue into just
-one part of your application
+يمكنك توصيل Vue في جزء واحد فقط من تطبيقك
```
Comment (Ma7eer) [inline assets/why-vue/arabic.js.srt]: I think 15 and 16 need to be rewritten. Here is a suggestion: 
يمكنك وصل vue الى جزء واحد من تطبيقك الذي قد يحتاج الى تجربة اكثر ثراء وتفاعلا

**Comment by Ma7eer** [comment]
Hello I added some comments but before I continue and make a mess was wondering if this is the type of thing you were looking for? or do you want me to "start a review"?

**Comment by sdras** [comment]
From my understanding, I think it would be ok to go change the content, but @chadidi please correct me if I'm mistaken

File: assets/why-vue/arabic.js.srt (hunk 1 above)
Comment (dtheb) [reply assets/why-vue/arabic.js.srt]: I think also ودود doesn't really fit, not sure what word to use without rephrasing the sentences

**Comment by dtheb** [comment]
What's the main goal of the review? understanding (tech scene dialect) or grammars/classical Arabic?
Because these 2 styles can't really co-exist 😞

File: assets/why-vue/arabic.js.srt (hunk 3 above)
Comment (abdumu) [reply assets/why-vue/arabic.js.srt]: خادوم/خادم هي الكلمة الأفصح، سيرفر كلمة انجليزية.

**Comment by chadidi** [comment]
@Ma7eer it's gonna be nice from your part if you can fix this translation.

@sdras exactly, I am not very good with translations.

@dtheb the goal is to translate this script to "[Modern Standard Arabic](https://en.wikipedia.org/wiki/Modern_Standard_Arabic)".

Thanks.

File: assets/why-vue/arabic.js.srt (hunk 1 above)
Comment (abdumu) [reply assets/why-vue/arabic.js.srt]: Approachable: يسر التعامل
Performant: فعال وعملي (together)

**Comment by meabed** [comment]
@HashemKhalifa could you have a look?

File: assets/why-vue/arabic.js.srt [hunk 5]
```diff
@@ -0,0 +1,544 @@
...
+
+2
+00:00:02,418 --> 00:00:05,600
+أكثر ديناميكية وقوة
+بفضل جافا سكريبت.
+
+3
+00:00:05,600 --> 00:00:07,541
+لقد قمنا بنقل الكثير من
+الرموز التي كانت عادة في
```
Comment (HashemKhalifa) [inline assets/why-vue/arabic.js.srt]: لقد قمنا بنقل الكثير من الأكواد التي كانت عادة في

File: assets/why-vue/arabic.js.srt [hunk 6]
```diff
@@ -0,0 +1,544 @@
...
+
+4
+00:00:07,541 --> 00:00:10,120
+جهة الخادم إلى متصفحاتنا،
```
Comment (HashemKhalifa) [inline assets/why-vue/arabic.js.srt]: الخادم إلي المتصفح

File: assets/why-vue/arabic.js.srt [hunk 7]
```diff
@@ -0,0 +1,544 @@
...
+والمزيد من المطورين
+
+9
+00:00:19,781 --> 00:00:23,948
+أطر عمل جافا سكريبت
+مثل Angular أو React أو Vue.
+
+10
+00:00:24,821 --> 00:00:27,829
+Vue ودود،
```
Comment (HashemKhalifa) [inline assets/why-vue/arabic.js.srt]: ودود is not the proper word to use here

**Comment by AhmedAbdulrahman** [comment]
Well, it's a bit challenging to translate technical terms from English into Arabic and not easy to apply common translation standards for Arabic, you may find three or more different translations for the same source piece of text. I think It would be better if you provide some explanation alongside translation for complex terms. @sdras

**Comment by AhmedHelalAhmed** [comment]
i can contribute

**Comment by chadidi** [comment]
@AhmedAbdulrahman The goal is that for people who don't understand English to understand the video from the Arabic subtitles that we are working on, and for the technical terms mostly `javascript`, so it doesn't matter if it translated or not most people that are going to watch this video are developers or learning development and there is a high chance that they already know this terms.

**Comment by chadidi** [comment]
@AhmedHelalAhmed @Ma7eer @dtheb @meabed @abdumu We have this public community on transifex maybe it's going to be easier on you guys to translate and review this file their https://www.transifex.com/laravelarab/public/

is that ok @sdras 

Thanks

**Comment by sdras** [comment]
Hey @chadidi  @AhmedHelalAhmed @Ma7eer @dtheb @meabed @abdumu, how is this coming? No worries if you need more time, just checking in.

File: assets/why-vue/arabic.js.srt [hunk 8]
```diff
@@ -0,0 +1,544 @@
...
+00:01:19,460 --> 00:01:21,200
+كما هو الحال مع العديد من تطبيقات JavaScript،
+
+34
+00:01:21,200 --> 00:01:24,272
+نبدأ من الحاجة إلى عرض البيانات على صفحة الويب الخاصة بنا.
+
+35
+00:01:24,272 --> 00:01:26,591
+مع Vue، يبدأ الأمر بسهولة تامة.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
وهو أمر بغاية السهولة مع إطار Vue.
```

File: assets/why-vue/arabic.js.srt [hunk 9]
```diff
@@ -0,0 +1,544 @@
...
+
+36
+00:01:26,591 --> 00:01:29,745
+نحن نقوم بتضمين مكتبة Vue، وإنشاء نسخة Vue،
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
حيث نقوم بتضمين مكتبة Vue، ومن ثم نكتب كود تشغيلها 
```

File: assets/why-vue/arabic.js.srt [hunk 10]
```diff
@@ -0,0 +1,544 @@
...
+
+37
+00:01:29,745 --> 00:01:33,673
+وقم بتوصيل العنصر الجذري بمعرّف التطبيق.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
ونضع الآي دي (مُعرف العنصر) لأي عنصر هتمل نريده كأساس للعرض فيه.
```

File: assets/why-vue/arabic.js.srt [hunk 11]
```diff
@@ -0,0 +1,544 @@
...
+
+38
+00:01:33,673 --> 00:01:36,032
+EL تقف على العنصر.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
EL تعني عُنصر وهي اختصار لكلمة element الإنجليزية
```

File: assets/why-vue/arabic.js.srt [hunk 12]
```diff
@@ -0,0 +1,544 @@
...
+
+39
+00:01:36,032 --> 00:01:39,505
+سنقوم أيضًا بنقل بياناتنا داخل كائن
+
+40
+00:01:39,505 --> 00:01:41,993
+وتغيير X إلى تعبير
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
وتغيير X إلى أي صيغة برمجية
```

File: assets/why-vue/arabic.js.srt [hunk 13]
```diff
@@ -0,0 +1,544 @@
...
+
+41
+00:01:41,993 --> 00:01:44,544
+مع الأقواس المزدوجة المتعرجة.
+
+42
+00:01:44,544 --> 00:01:46,945
+كما ترون، إنه يعمل.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
هاهو يعمل كما ترى
```

File: assets/why-vue/arabic.js.srt [hunk 14]
```diff
@@ -0,0 +1,544 @@
...
+
+43
+00:01:46,945 --> 00:01:50,695
+رائع جدًا ، لكن سحر Vue يبدأ عندما تتغير البيانات.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
شي رائع! ولكن روعة إطار Vue تبدأ عندما تتغير وتتبدل البيانات.
```

File: assets/why-vue/arabic.js.srt [hunk 15]
```diff
@@ -0,0 +1,544 @@
...
+
+44
+00:01:50,695 --> 00:01:51,975
+إذا قفزنا إلى وحدة التحكم ،
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
لو ذهبنا لسطر الأوامر
```

File: assets/why-vue/arabic.js.srt [hunk 16]
```diff
@@ -0,0 +1,544 @@
...
+
+45
+00:01:51,975 --> 00:01:54,735
+نحن نغير قيمة منتجنا.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
وغيرنا قيمة المنتج
```

File: assets/why-vue/arabic.js.srt [hunk 17]
```diff
@@ -0,0 +1,544 @@
...
+00:01:54,735 --> 00:01:57,324
+لاحظ أنه بمجرد تغير قيمة المنتج ،
+
+47
+00:01:57,324 --> 00:01:59,953
+قام Vue بتحديث HTML الخاص بنا تلقائيًا.
+
+48
+00:01:59,953 --> 00:02:01,745
+هذا لأن Vue هو رد الفعل ،
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
هذا لأن Vue إطار تفاعلي
```

File: assets/why-vue/arabic.js.srt [hunk 18]
```diff
@@ -0,0 +1,544 @@
...
+
+49
+00:02:01,745 --> 00:02:03,273
+وهذا يعني أنه عندما تتغير بياناتنا ،
+
+50
+00:02:03,273 --> 00:02:05,193
+يهتم Vue بتحديث جميع الأماكن
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
فإن إطار Vue سيحدث كافة الأماكن المتغيرة بنفسه دون تدخل
```

File: assets/why-vue/arabic.js.srt [hunk 19]
```diff
@@ -0,0 +1,544 @@
...
+
+51
+00:02:05,193 --> 00:02:07,255
+نحن نستخدمها في صفحة الويب الخاصة بنا.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
في صفحة الويب الخاصة بنا.
```

File: assets/why-vue/arabic.js.srt [hunk 20]
```diff
@@ -0,0 +1,544 @@
...
+
+52
+00:02:07,255 --> 00:02:10,305
+هذا يعمل مع أي نوع من البيانات ، وليس فقط سلاسل.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
وهذا الأمر يعمل مع كافة أنواع البيانات وليس فقط النصوص.
```

File: assets/why-vue/arabic.js.srt [hunk 21]
```diff
@@ -0,0 +1,544 @@
...
+00:02:11,604 --> 00:02:14,604
+دعونا نستخدم مجموعة من المنتجات.
+
+55
+00:02:14,604 --> 00:02:17,865
+ثم سنقوم بتحديث H2 إلى قائمة غير مرتبة ،
+
+56
+00:02:17,865 --> 00:02:20,535
+وإنشاء عنصر قائمة جديد لكل منتج في الصفيف
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
وإنشاء عنصر قائمة جديد لكل منتج في المصفوفة
```

File: assets/why-vue/arabic.js.srt [hunk 22]
```diff
@@ -0,0 +1,544 @@
...
+
+57
+00:02:20,535 --> 00:02:22,618
+باستخدام التوجيه Vue للتوجيه.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
باستخدام التوجيه (الصيغة البرمجية) v-for.
```

File: assets/why-vue/arabic.js.srt [hunk 23]
```diff
@@ -0,0 +1,544 @@
...
+00:02:24,353 --> 00:02:26,615
+بهذه الطريقة ، يحصل كل منتج على عنصر قائمة خاص به
+
+59
+00:02:26,615 --> 00:02:28,495
+كما ترون هنا.
+
+60
+00:02:28,495 --> 00:02:30,352
+هذا لا يزال مفتعلة بعض الشيء على الرغم من.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
لكن هذا الشيء يبدو مختلق ذو بيانات معدة مسبقاً بعض الشيء
```

File: assets/why-vue/arabic.js.srt [hunk 24]
```diff
@@ -0,0 +1,544 @@
...
+
+61
+00:02:30,352 --> 00:02:32,312
+لذلك دعونا نبدأ بقائمتنا فارغة
+
+62
+00:02:32,312 --> 00:02:34,815
+وجلب منتجاتنا من واجهة برمجة التطبيقات الفعلية
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
ومن ثم جلب منتجاتنا من واجهة برمجية حقيقية
```

File: assets/why-vue/arabic.js.srt [hunk 25]
```diff
@@ -0,0 +1,544 @@
...
+00:02:37,433 --> 00:02:38,863
+عندما يتم إنشاء التطبيق لدينا ،
+
+65
+00:02:38,863 --> 00:02:41,866
+سنجلب أحدث المنتجات من واجهة برمجة التطبيقات هذه.
+
+66
+00:02:41,866 --> 00:02:43,199
+كل ما عليك أن تأخذ منه هو ذلك
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
كل ماعليك معرفته هنا هو أننا
```

File: assets/why-vue/arabic.js.srt [hunk 26]
```diff
@@ -0,0 +1,544 @@
...
+
+67
+00:02:43,199 --> 00:02:45,063
+نحن نحضر قائمتنا من المنتجات
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
نجلب قائمتنا من المنتجات
```

File: assets/why-vue/arabic.js.srt [hunk 27]
```diff
@@ -0,0 +1,544 @@
...
+
+68
+00:02:45,063 --> 00:02:46,923
+وتحديثها في بياناتنا.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
ومن ثم تحديثها في بياناتنا
```

File: assets/why-vue/arabic.js.srt [hunk 28]
```diff
@@ -0,0 +1,544 @@
...
+00:02:46,923 --> 00:02:48,281
+كما ترون على صفحة الويب لدينا ،
+
+70
+00:02:48,281 --> 00:02:51,435
+كل عنصر قائمة يعرض الكائن الذي تم إرجاعه.
+
+71
+00:02:51,435 --> 00:02:53,474
+حسنا ، إنه ليس من الممكن قراءة الإنسان بعد ،
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
حسنا ، يبدو أنه غير قابل للقراءة بعد ،
```

File: assets/why-vue/arabic.js.srt [hunk 29]
```diff
@@ -0,0 +1,544 @@
...
+
+72
+00:02:53,474 --> 00:02:56,265
+لذلك دعونا تغيير الطريقة التي يتم عرضها.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
لذلك دعونا نغير الطريقة التي يتم عرضها فيها.
```

File: assets/why-vue/arabic.js.srt [hunk 30]
```diff
@@ -0,0 +1,544 @@
...
+
+73
+00:02:56,265 --> 00:02:59,643
+سنقوم بطباعة الكمية والاسم.
+
+74
+00:02:59,643 --> 00:03:02,265
+وهناك ، طبع بشكل جيد.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
الآن، قد تم عرضه بشكل جيد.
```

File: assets/why-vue/arabic.js.srt [hunk 31]
```diff
@@ -0,0 +1,544 @@
...
+00:03:02,265 --> 00:03:03,563
+قد نرغب في لفت الانتباه إلى
+
+76
+00:03:03,563 --> 00:03:05,675
+العناصر التي تحتوي على كمية صفر.
+
+77
+00:03:05,675 --> 00:03:09,595
+لذلك دعونا نضيف القليل من الامتداد مع النص ، من المخزون.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
لذلك دعونا نضيف عنصر عرض يحمل النص: منتهي من المخزون.
```

File: assets/why-vue/arabic.js.srt [hunk 32]
```diff
@@ -0,0 +1,544 @@
...
+
+78
+00:03:09,595 --> 00:03:13,334
+يجب أن يظهر هذا فقط إذا كانت كمية العنصر تساوي صفرًا.
+
+79
+00:03:13,334 --> 00:03:16,134
+سنستخدم التعليمة v-if.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
سنستخدم التوجيه v-if.
```

File: assets/why-vue/arabic.js.srt [hunk 33]
```diff
@@ -0,0 +1,544 @@
...
+
+80
+00:03:16,134 --> 00:03:20,362
+لأن سترة لدينا لديها كمية من الصفر ، انها من المخزون.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
لأن السترة لدينا كميتها صفر فهي: منتهي من المخزون.
```

File: assets/why-vue/arabic.js.srt [hunk 34]
```diff
@@ -0,0 +1,544 @@
...
+00:03:20,362 --> 00:03:22,081
+ماذا لو أردنا طباعة الرقم الإجمالي
+
+82
+00:03:22,081 --> 00:03:24,641
+من المنتجات تحت قائمتنا؟
+
+83
+00:03:24,641 --> 00:03:26,468
+لجعل هذا العمل ، نحتاج إلى إنشاء
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
لفعل ذلك نحتاج إلى إنشاء
```

File: assets/why-vue/arabic.js.srt [hunk 35]
```diff
@@ -0,0 +1,544 @@
...
+
+84
+00:03:26,468 --> 00:03:29,570
+خاصية محسوبة تسمى totalProducts
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
خاصية محسوبة (computed property) ونسميها totalProducts
```

File: assets/why-vue/arabic.js.srt [hunk 36]
```diff
@@ -0,0 +1,544 @@
...
+
+85
+00:03:29,570 --> 00:03:34,228
+الذي يعيد الكمية الإجمالية لمنتجاتنا.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
ونجعلها تعيد الكمية الإجمالية لمنتجاتنا.
```

File: assets/why-vue/arabic.js.srt [hunk 37]
```diff
@@ -0,0 +1,544 @@
...
+
+86
+00:03:34,228 --> 00:03:36,490
+إذا كنت غير معتاد على استخدام جافا سكريبت ،
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
إذا كنت لاتعرف دالة reduce في لغة جافا سكريبت ،
```

File: assets/why-vue/arabic.js.srt [hunk 38]
```diff
@@ -0,0 +1,544 @@
...
+
+87
+00:03:36,490 --> 00:03:40,468
+انها تضيف كل الكميات من كل منتج.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
فهي تجمع كل الكميات لكل منتج.
```

File: assets/why-vue/arabic.js.srt [hunk 39]
```diff
@@ -0,0 +1,544 @@
...
+
+88
+00:03:40,468 --> 00:03:41,851
+كما ترون ، في متصفحنا ،
+
+89
+00:03:41,851 --> 00:03:45,308
+انها تضيف بشكل صحيح حتى جميع الكميات.
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
فهي أضافت بشكل صحيح جميع الكميات.
```

File: assets/why-vue/arabic.js.srt [hunk 40]
```diff
@@ -0,0 +1,544 @@
...
+
+90
+00:03:45,308 --> 00:03:47,588
+هنا في المتصفح ، أريد أن أظهر لك Vue
```
Comment (abdumu) [inline assets/why-vue/arabic.js.srt]:
```suggestion
هنا في المتصفح ، أريد أن اريك إضافة Vue
```

File: assets/why-vue/arabic.js.srt (hunk 5 above)
[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: @abdumu The translation is not correct you have to translate from this English file: https://www.transifex.com/laravelarab/vuejs-website/translate/#ar/englishjssrt/154471465

File: assets/why-vue/arabic.js.srt (hunk 6 above)
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: @HashemKhalifa The English version says `the server side into our browsers,`

File: assets/why-vue/arabic.js.srt [hunk 41]
```diff
@@ -0,0 +1,544 @@
...
+يساعدك على إنشاء
+
+12
+00:00:29,860 --> 00:00:33,381
+قاعدة شفرة أكثر قابلية للإصلاح وللإختبار.
+
+13
+00:00:33,381 --> 00:00:36,229
+Vue هو إطار عمل جافا سكريبت تقدمي،
+مما يعني،
```
[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: Missing comma

File: assets/why-vue/arabic.js.srt [hunk 42]
```diff
@@ -0,0 +1,544 @@
...
+يعمل على الخادم،
+
+15
+00:00:38,621 --> 00:00:41,627
+يمكنك توصيل Vue
+في جزء من تطبيقك
+
+16
+00:00:41,627 --> 00:00:44,381
+الذي يحتاج إلى تجربة أكثر
```
[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: @abdumu 
the English version
```
that needs a richer, more
interactive experience.
```

File: assets/why-vue/arabic.js.srt [hunk 43]
```diff
@@ -0,0 +1,544 @@
...
+00:00:44,381 --> 00:00:46,269
+أو، إذا كنت ترغب في بناء المزيد من منطق الأعمال
+
+18
+00:00:46,269 --> 00:00:48,240
+إلى واجهتك الأمامية من الحصول على الذهاب ،
+
+19
+00:00:48,240 --> 00:00:50,611
+لدى Vue المكتبات الأساسية والنظام البيئي
```
[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: English
```
Vue has the core libraries
and the ecosystem
```
Arabic
```
فلدى Vue المكتبات الأساسية
و النظام المتكامل
```

File: assets/why-vue/arabic.js.srt [hunk 44]
```diff
@@ -0,0 +1,544 @@
...
+
+20
+00:00:50,611 --> 00:00:52,421
+ستحتاج إلى مقياس.
```
[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: `الذي تحتاج للتوسع.`

File: assets/why-vue/arabic.js.srt [hunk 45]
```diff
@@ -0,0 +1,544 @@
...
+
+21
+00:00:52,421 --> 00:00:53,829
+مثل غيرها من أطر الواجهة الأمامية،
```
[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: English
`Like other frontend frameworks,`
Arabic
`مثل غيرها من أطر الواجهة الأمامية،`

File: assets/why-vue/arabic.js.srt [hunk 46]
```diff
@@ -0,0 +1,544 @@
...
+
+22
+00:00:53,829 --> 00:00:55,960
+تسمح لك Vue بأخذ صفحة ويب وتقسيمها
```
[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: English
```
Vue allows you to take a
web page and split it up
```
Arabic
```
تسمح لك Vue بأخذ
صفحة ويب وتقسيمها
```

File: assets/why-vue/arabic.js.srt [hunk 47]
```diff
@@ -0,0 +1,544 @@
...
+00:01:00,000 --> 00:01:02,291
+يجب تقديم CSS، وجافا سكريبت
+
+25
+00:01:02,291 --> 00:01:04,269
+تلك القطعة من الصفحة.
+
+26
+00:01:04,269 --> 00:01:06,260
+التالي، سنلقي نظرة على Vue عن كثب
```
[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]:
```
Next, we'll take a look
at Vue in action by
```

File: assets/why-vue/arabic.js.srt (hunk 5 above)
[the remark this answers is not in the document]
Comment (abdumu) [reply assets/why-vue/arabic.js.srt]: لا يجب أن تكون مطابقة بل مفهومة، حاول قراءتها مع مايجيء بعدها.

[the remark this answers is not in the document]
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: @abdumu أتفق معك ولكن أظن أنها مفهومة 

```
لقد قمنا بنقل الكثير من
الأكواد التي كانت عادة في
جهة الخادوم إلى متصفحاتنا،
```

File: assets/why-vue/arabic.js.srt (hunk 12 above)
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]: `and change X into an expression` 

`و تغيير X إلى صيغة برمجية`

File: assets/why-vue/arabic.js.srt (hunk 31 above)
Comment (chadidi) [reply assets/why-vue/arabic.js.srt]:
```
لذلك دعونا نضيف وسم span
يحمل النص، منتهي من المخزون.
```

**Comment by chadidi** [comment]
Hey @AhmedHelalAhmed @Ma7eer @dtheb @meabed @abdumu can you guys check the translation one last time, and if you can try it with the video also.

Thanks

**Comment by chadidi** [comment]
Hey @sdras, I think this PR is ready to merge.

**Comment by sdras** [comment]
Ok thank you!

**Pull request merged**